docs/project/release-process.md

```diff
@@ -4,60 +4,27 @@
-2. The change log generator configuration below will look for unreleased changes on a specific branch. The branch will be `master` for a major/minor release, or a release branch (`v0.4-branch`) for a patch release. You will need to set the branch using the `--release-branch` argument.
-3. You should also set the `--future-release` argument. This is the version you are releasing. The version can still be changed at a later date. 
-4. Update the arguments below and run the command to generate the change log to the console.
-
-```
-docker run -it --rm ferrarimarco/github-changelog-generator \
---user feast-dev \
---project feast  \
---release-branch <release-branch-to-find-changes>  \
---future-release <proposed-release-version>  \
---unreleased-only  \
---no-issues  \
---bug-labels kind/bug  \
---enhancement-labels kind/feature  \
---breaking-labels compat/breaking  \
--t <your-github-token>  \
---max-issues 1 \
--o
-```
-
-1. Review each change log item.
-   * Make sure that sentences are grammatically correct and well formatted (although we will try to enforce this at the PR review stage). 
-   * Make sure that each item is categorised correctly. You will see the following categories: `Breaking changes`, `Implemented enhancements`, `Fixed bugs`, and `Merged pull requests`. Any unlabelled PRs will be found in `Merged pull requests`. It's important to make sure that any `breaking changes`, `enhancements`, or `bug fixes` are pulled up out of `merged pull requests` into the correct category. Housekeeping, tech debt clearing, infra changes, or refactoring do not count as `enhancements`. Only enhancements a user benefits from should be listed in that category.
-   * Make sure that the "Full Change log" link is actually comparing the correct tags (normally your released version against the previously version).
-   * Make sure that release notes and breaking changes are present.
-
-### Flag Breaking Changes & Deprecations
-
-It's important to flag breaking changes and deprecation to the API for each release so that we can maintain API compatibility.
-
-Developers should have flagged PRs with breaking changes with the `compat/breaking` label. However, it's important to double check each PR's release notes and contents for changes that will break API compatibility and manually label `compat/breaking` to PRs with undeclared breaking changes. The change log will have to be regenerated if any new labels have to be added.
+### Pre-release Verification (Verification that wheels are built correctly) for minor release.
+1. Merge upstream master changes into your **fork**. Make sure you are running the workflow off of your fork!
+2. Create a tag manually for the release on your fork. For example, if you are doing a release for version 0.22.0, create a tag by doing the following.
+   - Checkout master branch and run `git tag v0.22.0`.
+   - Run `git push --tags` to push the tag to remote.
```

Comment:
To push the tag to your forks master branch.

Reply:
Fixed